### PR TITLE
Master

### PR DESCRIPTION
**Pull request by yeja94**
oops not Dealer, I meant Game

### CODE REVIEW

File: src/test/java/model/testCards.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    
+    @Test
+    public void testSuit(){
+        Cards c = new Cards(7,Suit.Hearts);
+        assertEquals(Suit.Hearts,c.getSuit());
+    }
+    @Test
+    public void testString(){
+        Cards c = new Cards(7, Suit.Hearts);
+        assertEquals("7Hearts",c.toString());
```
Comment (mandrasigma) [inline src/test/java/model/testCards.java:21]: Would you want to split the num and word w a space to make parsing easier?

File: src/test/java/model/testGame.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    @Test
+    public void testGameCreation(){
+        Game g = new Game();
+        assertNotNull(g);
+    }
+    @Test
+    public void testGameInitialize(){
+        Game g = new Game();
+        g.shuffle();
+        assertNotEquals(2,g.deck.get(0).getValue());
```
Comment (mandrasigma) [inline src/test/java/model/testGame.java:20]: Does this mean there's a 1 in 13 chance this test will fail upon compilation?

File: src/test/java/model/testGame.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    @Test
+    public void testBuildDeck(){
+        Game g = new Game();
+        g.buildDeck();
+        assertEquals(48,g.deck.size());
```
Comment (mandrasigma) [inline src/test/java/model/testGame.java:26]: After the player and dealer have been given cards?

File: src/test/java/model/testGame.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    @Test
+    public void testGameStart(){
```
Comment (mandrasigma) [inline src/test/java/model/testGame.java:29]: Assertion? Would it be 48 while the previous one should be 52?

Comment (yeja94) [inline src/test/java/model/testGame.java:29]: 48 after dealing 4 cards, 2 for player and 2 for dealer

**Pull request closed without merging**